src/lein_monolith/plugin.clj

```diff
@@ -139,3 +139,13 @@
         (reduce-kv add-active-profile project profiles)))
     ; Normal project, don't activate.
     project))
+
+
+(defn add-middleware
+  "Update the given project to include the plugin middleware. Appends the
+  middleware symbol if it is not already present."
+  [project]
+  (let [mw-sym 'lein-monolith.plugin/middleware]
+    (if (some #{mw-sym} (:middleware project))
+      project
+      (update project :middleware (fnil into []) [mw-sym]))))
```

Comment:
Any reason you use `(fnil into [])` and wrap the `mw-sym` with a vector instead of `(update project :middleware (fnil conj []) mw-sym)`?

Reply:
Hah nope, my brain just jumped to `into` for some reason.